src/functions/genfrac.js

```diff
@@ -374,7 +397,7 @@ defineFunction({
     names: ["\\genfrac"],
     props: {
         numArgs: 6,
-        greediness: 6,
```

Comment:
`allowedInArgument` is easier to understand than greediness especially given that we had to give it different numbers to work properly.